roles/os_hardening/tasks/cron.yml

```diff
@@ -0,0 +1,84 @@
+---
+# Granting write access to this directory for non-privileged users could provide
+# them the means for gaining unauthorized elevated privileges.
+# Granting read access to this directory could give an unprivileged user insight
+# in how to gain elevated privileges or circumvent auditing controls.
+# CIS 5.1.2 - CIS 5.1.7
+#
+- name: Check if /etc/crontab exists
+  stat:
+    path: /etc/crontab
+  register: osh_crontab
+
+- name: Ensure permissions on /etc/crontab are configured
+  ansible.builtin.file:
+    path: /etc/crontab
+    owner: root
+    group: root
+    mode: '0600'
+  when: osh_crontab.stat.exists
+
+- name: Check if /etc/cron.hourly exists
+  stat:
+    path: /etc/cron.hourly
+  register: osh_cron_hourly
+
+- name: Ensure permissions on /etc/cron.hourly are configured
+  ansible.builtin.file:
+    path: /etc/cron.hourly
+    owner: root
+    group: root
+    mode: '0600'
```

Comment:
Normally you'd need execute-permissions to enter the directory, but root doesn't. To stick to the convention let's change that to `0700` on directories.

Reply:
This was a stupid copy-paste error. I should have noticed it. I changed it to "mode: og-rwx", which I think is better. It shows better what we want to achieve.